src/SmartComponents/BaselinesPage/CreateBaselineModal/CreateBaselineModal.js

```diff
@@ -14,12 +14,27 @@ class CreateBaselineModal extends Component {
         this.submitBaselineName = this.submitBaselineName.bind(this);
 
         this.state = {
-            baselineName: ''
+            baselineName: '',
+            check1: false,
+            check2: false,
+            check3: false
```

Comment:
I know this is a kinda POC, but if its also the starter work for this change, I would ask we consider giving these checks names that are meaningful.